src/app/campaign-details/campaign-details.component.html

```diff
@@ -25,6 +25,22 @@ <h3>Campaign Targets</h3>
     <p>Donations: {{ campaign.donationCount }}</p>
   </div>
 
+  <div *ngIf="campaign.budgetDetails.length != 0">
```

Comment:
@tbg-jasonfung I think these would be a little more readable and less likely to ever have weirdness with type coercion if they checked specifically for `> 0`